packages/metal-soy/src/Soy.js

```diff
@@ -59,6 +61,11 @@ class Soy extends IncrementalDomRenderer.constructor {
 				data[params[i]] = component[params[i]].bind(component);
 			}
 		}
+
+		if (component.getTemplateData) {
```

Comment:
@mairatma should `isFn` be used here? Is usually in Metal the presence of a property defined by the contract enough?